groups/infrastructure/profiles/staging-eu-west-2/staging/vars

```diff
@@ -0,0 +1,14 @@
+aws_bucket = "staging-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-service-staging-terraform-state"
+environment = "staging"
+deploy_to = "staging"
+state_prefix = "env:/staging"
```

Comment:
See https://github.com/companieshouse/file-transfer-stack/pull/2/files#r1895766757

Reply:
This is fixed now. Commit is pushed.